src/frontend/komponenter/Fagsak/Behandlingsresultat/Behandlingsresultat.tsx

```diff
@@ -229,6 +266,24 @@ const Behandlingsresultat: React.FunctionComponent<IBehandlingsresultatProps> =
             {åpenBehandling.endretUtbetalingAndeler.length > 0 && (
                 <EndretUtbetalingAndelTabell åpenBehandling={åpenBehandling} />
             )}
+            {toggles[ToggleNavn.overgangsordningErTilgjengelig] &&
+                åpenBehandling.årsak == BehandlingÅrsak.OVERGANGSORDNING_2024 && (
+                    <>
+                        <OvergangsordningAndelTabell åpenBehandling={åpenBehandling} />
```

Comment:
Vi ønsker kun å rendre `OvergangsordningAndelTabell` dersom det finnes `overgangsordningAndeler`. Typ: 
```suggestion
                        {åpenBehandling.overgangsordningAndeler.length > 0 && (<OvergangsordningAndelTabell åpenBehandling={åpenBehandling} /> )}
```

Reply:
I en behandling med årsak `OVERGANGSORDNING_2024` gir det ikke mening at den er tom, så beholder den der, selv om den opprinnelig er tom, men enig i at den kan skjules i senere behandlinger